__tests__/models/StreetParkingRulesSpec.js

```diff
@@ -29,28 +31,46 @@ describe("StreetParkingRules", () => {
     describe('when parking Monday-Saturday after 7pm', () => {
       describe('and car is on wrong side of street', () => {
         it('returns 0', () => {
-          const rules = new StreetParkingRules()
           const wednesdayEvening = new Date(2019, 3, 3, 19, 1)
           expect(rules.timeRemainingOnSide(wednesdayEvening, 'odd')).toEqual(0)
         })
       })
       describe('and car is on even side of street on a Wednesday', () => {
         it('returns minutes until switch', () => {
-          const rules = new StreetParkingRules()
           const aWednesdayMorning = new Date(2019, 3, 3, 19, 1)
           expect(rules.timeRemainingOnSide(aWednesdayMorning, 'even')).toEqual(1439 * 60)
         })
       })
       describe('and car is on odd side of street on a Tuesday', () => {
         it('returns minutes until switch', () => {
-          const rules = new StreetParkingRules()
           const aWednesdayMorning = new Date(2019, 3, 2, 19, 1)
           expect(rules.timeRemainingOnSide(aWednesdayMorning, 'odd')).toEqual(1439 * 60)
         })
       })
     })
     describe('when parking on Sunday', () => {
+      describe('and car is on odd side of the street', () => {
+        it('returns minutes until switch', () => {
+          const sundayNoon = new Date(2019, 3, 21, 12, 1)
```

Comment:
I think I was imagining an 8pm test for this case, b/c that's a part of what's different... maybe it's covered by the one below.